src/components/Send.jsx

```diff
@@ -425,6 +426,17 @@ export default function Send() {
     return () => abortCtrl.abort()
   }, [isOperationDisabled, wallet, reloadCurrentWalletInfo, reloadServiceInfo, loadConfigValue, t])
 
+  useEffect(() => {
+    if (walletInfo?.addressSummary[destination]) {
```

Comment:
```suggestion
    if (destination && walletInfo?.addressSummary[destination]) {
```
?

Apparently..
```
var dict = { null: 1 }
dict[null]
// yields 1
```

Reply:
Yeah but will `addressSummary` ever have a `null` entry we'd accidentally access that way?

Reply:
I guess not, just trying to come up with stuff that breaks things. Maybe in the future "null" is a valid Bitcoin invoice? : D
Chances are for that are basically zero, yes. I'd say it is just good practice (apparently, `{"null": 1}[null]` yields also `1`).
I am with you, though, it's a nit.

Reply:
Added the null check, you're right it's good practice.

> Maybe in the future "null" is a valid Bitcoin invoice? : D

If that'd be the case, then we'd want to check if the `null` address is reused though right? :D

Reply:
> If that'd be the case, then we'd want to check if the `null` address is reused though right? :D
`null` is falsy, `"null"` is truthy, oh my..  : D